src/Shared/Utilities.ts

```diff
@@ -1,2 +1,6 @@
 export const UTF_PIPE = '|';
 export const UTF_COLON = ':';
+export const STYLE_BOX_VALUES = {
```

Comment:
Q: Just double-checking: are X's and Y's are exactly the same for both, BondStyleBox and EquityStyleBox?